servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientBuilder.java

```diff
@@ -50,72 +58,235 @@
  * @param <U> the type of address before resolution (unresolved address)
  * @param <R> the type of address after resolution (resolved address)
  */
-public abstract class GrpcClientBuilder<U, R>
-        implements SingleAddressGrpcClientBuilder<U, R, ServiceDiscovererEvent<R>> {
+public abstract class GrpcClientBuilder<U, R> {
 
     private boolean appendedCatchAllFilter;
 
-    @Override
+    /**
+     * Sets the {@link IoExecutor} for all clients created from this builder.
+     *
+     * @param ioExecutor {@link IoExecutor} to use.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> ioExecutor(IoExecutor ioExecutor);
 
-    @Override
+    /**
+     * Sets the {@link BufferAllocator} for all clients created from this builder.
+     *
+     * @param allocator {@link BufferAllocator} to use.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> bufferAllocator(BufferAllocator allocator);
 
-    @Override
+    /**
+     * Sets the {@link GrpcExecutionStrategy} for all clients created from this builder.
+     *
+     * @param strategy {@link GrpcExecutionStrategy} to use.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> executionStrategy(GrpcExecutionStrategy strategy);
 
-    @Override
+    /**
+     * Add a {@link SocketOption} for all clients created by this builder.
+     *
+     * @param option the option to apply.
+     * @param value the value.
+     * @param <T> the type of the value.
+     * @return {@code this}.
+     * @see StandardSocketOptions
+     * @see ServiceTalkSocketOptions
+     */
     public abstract <T> GrpcClientBuilder<U, R> socketOption(SocketOption<T> option, T value);
 
-    @Override
+    /**
+     * Enables wire-logging for connections created by this builder.
+     *
+     * @param loggerName The name of the logger to log wire events.
+     * @param logLevel The level to log at.
+     * @param logUserData {@code true} to include user data (e.g. data, headers, etc.). {@code false} to exclude user
+     * data and log only network events.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> enableWireLogging(String loggerName, LogLevel logLevel,
                                                               BooleanSupplier logUserData);
 
-    @Override
+    /**
+     * Configurations of various underlying protocol versions.
+     * <p>
+     * <b>Note:</b> the order of specified protocols will reflect on priorities for ALPN in case the connections are
+     * secured.
+     *
+     * @param protocols {@link HttpProtocolConfig} for each protocol that should be supported.
+     * @return {@code this}.
+     */
     public abstract GrpcClientBuilder<U, R> protocols(HttpProtocolConfig... protocols);
```

Comment:
Consider adding `@see HttpProtocolConfigs` tag

Reply:
For that to be visible, the grpc-api module would need to depend on http-netty. Do we want that?

Reply:
No, that was overlook, I thought it's in api